hist/hist/src/TF1Convolution.cxx

```diff
@@ -90,11 +90,14 @@ void TF1Convolution::InitializeDataMembers(TF1* function1, TF1* function2, Bool_
    fFunction1->SetBit(TF1::kNotGlobal, kTRUE);
    fFunction2->SetBit(TF1::kNotGlobal, kTRUE);
 
-   // add by default an extra 10% on  each side
+   // use by default range of first function 
    fFunction1->GetRange(fXmin, fXmax);
-   Double_t range = fXmax - fXmin;
-   fXmin       -= 0.1*range;
-   fXmax       += 0.1*range;
+   // when using FFT add by default an extra 10% on  each side
+   if (useFFT) {
```

Comment:
Can we also use `SetExtraRange` for consistency here?
```
if (useFFT) SetExtraRange(0.1);
```

Reply:
Thank you for the suggestion about the extraRangeFactor. It will be good to implement it as a static class member.

Reply:
I have implemented in the new commit the above suggestions